bin/peas_shell

```diff
@@ -216,16 +219,35 @@ class PanSensorShell(cmd.Cmd):
             'cam_1': {'pin': 6, 'board': 'camera_board'},
         }
 
+        # NOTE: These are not pins but index numbers
+        powerboard_relay_lookup = {
+            'mount': {'pin': 0, 'board': 'power_board'},
+            'fan': {'pin': 1, 'board': 'power_board'},
+            'weather': {'pin': 2, 'board': 'power_board'},
+            'camera_box': {'pin': 3, 'board': 'power_board'},
+        }
+
+        if 'power_board' in self.environment.serial_readers:
+            relay_lookup = powerboard_relay_lookup
+        else:
+            relay_lookup = telemetry_relay_lookup
+
         try:
             relay_info = relay_lookup[relay]
-            self.environment.serial_readers[relay_info['board']]['reader'].write("{},9".format(relay_info['pin']))
+            serial_connection = self.environment.serial_readers[relay_info['board']]['reader']
+
+            serial_connection.ser.flushInput()
```

Comment:
@jamessynge I'm curious about your thoughts on this. It seemed to help in my testing yesterday. Before this toggle commands would sometimes lag (10+ seconds).